Dockerfile

```
@@ -7,6 +7,8 @@ COPY . /opt/shipwire-analytics-grabber
 WORKDIR  /opt/shipwire-analytics-grabber
 
 # Install python project, install modules
-RUN      pip3 install -r requirements.txt
+RUN      python3 -m ensurepip \
+         && pip3 install --upgrade pip \
+         && pip3 install -r requirements.txt
```

Weird align on the continuations here.

Do you mind adding in a space to be a little more consistent with the shell style in our other repos?
```suggestion
          && pip3 install --upgrade pip \
          && pip3 install -r requirements.txt
```